bin/root-transition.sh

```diff
@@ -151,19 +157,68 @@ transition() {
   echo "Please save them safely and privately."
 }
 
-case $1 in
-  check)
-    check
+check_version_namespace() {
+  local OUT=$(kubectl get po -n ${1} -o yaml | grep "istio\/proxyv2\:1\.")
+
+  for LINE in ${OUT};
+  do
+    if [[ ${LINE} == *"istio/proxyv2"* ]]; then
+      LINE=${LINE#"gke.gcr.io/istio/proxyv2:"};
+      LINE=${LINE#"docker.io/istio/proxyv2:"};
+      LINE=${LINE#"istio/proxyv2:"};
```

Comment:
I also observed gcr.io/gke-release/istio/proxyv2:1.1.10 during the test. Perhaps add "gcr.io/gke-release/istio/proxyv2:"?

Reply:
Sure.